web/src/app/globals.css

```
@@ -188,14 +188,84 @@ html {
+  margin: 1rem 0;
+}
+.report-prose table {
+  --d-table: table;
+  --d-thead: table-header-group;
+  --d-group: table-row-group;
+  --d-row: table-row;
+  --d-cell: table-cell;
+  --d-label: none;
+  --row-border: 0;
+  --cell-border-top: 1px solid var(--border);
+  display: var(--d-table);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)